Project/Gem/Source/Scripting/PayloadDespawnerComponent.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root
+ * of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Component/Component.h>
+#include <AzCore/Component/TickBus.h>
+#include <AzCore/RTTI/ReflectContext.h>
+#include <AzFramework/Physics/Common/PhysicsSimulatedBodyEvents.h>
+
+namespace ROS2::Demo
+{
+    using Tag = AZ::Crc32;
+
+    class PayloadDespawnerComponent
```

Comment:
It is extremelly specific controller, please add descriptive documentation here.